Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY requirements-pi.txt .
 COPY requirements.txt .
 RUN pip3 install -r requirements-pi.txt
 RUN apt-get update
-RUN apt-get install -y gpac
+RUN apt-get install -y gpac ffmpeg
```

Comment:
In order for this to take effect, it needs to be moved to the [CustomPiOS script](https://github.com/interactionresearchstudio/NaturewatchCameraServer/blob/c168da465a391ab4029ab0b64c02db4ee32ab6f8/os/modules/naturewatchcamera/start_chroot_script#L41). That way ffmpeg will be bundled into the GitHub Action that builds the downloadable Raspberry Pi image.

Reply:
Thanks, will do!

> Worth checking if anything else that needs an apt install needs to be added too!

How do I do such check? I'm not (yet) familiar with CustomPiOS.